src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -260,15 +267,15 @@ describe('handle-accessibility', () => {
     });
 
     it('should announce cell content and selection status for selected first cell after focusing on it', () => {
-      const tmpCell = global.document.createElement('td');
-      tmpCell.classList.add('selected');
+      const tmpCell = global.document.createElement('td') as HTMLDivElement;
```

Comment:
```suggestion
      const tmpCell = global.document.createElement('td');
```
I do not think that you need to convert the HTMLTableCellElement to HTMLDivElement here